src/Raven.Server/Config/Categories/CoreConfiguration.cs

```diff
@@ -48,6 +48,11 @@ public class CoreConfiguration : ConfigurationCategory
         [ConfigurationEntry("Setup.Mode", ConfigurationEntryScope.ServerWideOnly)]
         public SetupMode SetupMode { get; set; }
 
+        [Description("The URLs which the server should contact when requesting certificates from Let's Encrypt. If not defined, the Let's Encrypt production server will be used.")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("AcmeStagingUrl", ConfigurationEntryScope.ServerWideOnly)]
+        public string AcmeStagingUrl { get; set; }
```

Comment:
This is the url you need to enter for staging:
"https://acme-staging.api.letsencrypt.org/directory"